.github/actions/gke/k8s-versions.yaml

```diff
@@ -0,0 +1,22 @@
+# List of k8s version for GKE tests
+---
+k8s:
+  - version: "1.22"
+    zone: northamerica-northeast1-c
+    vmIndex: 1
+  - version: "1.23"
+    zone: europe-west6-b
+    vmIndex: 2
+  - version: "1.24"
+    zone: us-west2-a
+    vmIndex: 3
+    default: true
+  - version: "1.25"
+    zone: asia-northeast1-c
+    vmIndex: 4
+  - version: "1.26"
+    zone: europe-north1-b
+    vmIndex: 5
+  - version: "1.27"
+    zone: us-east5-a
+    vmIndex: 6
```

Comment:
[Nit, non-blocking] Let's try to make sure all the files end with an empty line for consistency (note: not repeating the nit but this also applies to some other files in the PR).

Reply:
Fixed it, and while at it discovered and fixed a schema violation as well.